hoot-rnd/src/main/cpp/hoot/rnd/scoring/MatchScoringMapPreparer.cpp

```diff
@@ -91,7 +91,7 @@ void MatchScoringMapPreparer::prepMap(OsmMapPtr map, const bool removeNodes)
   {
     // remove all REF1/REF2 tags from the nodes.
     RemoveTagsVisitor removeRef(MetadataTags::Ref1(), MetadataTags::Ref2());
-    IsNodeFilter nodeFilter(Filter::KeepMatches);
+    NodeCriterion nodeFilter;
```

Comment:
I know, I know.  It is nit-picky but shouldn't it now be `NodeCriterion nodeCriterion`? :smile:

Reply:
Yeah, it should.  I'll do that.  Did some of those but not others.  I sometimes use the terms criterion and filter interchangeably myself, but now that Filter is gone that shouldn't happen.

Reply:
I'm finding quite a few more like that.  Fixing them all.